.mailmap

```diff
@@ -847,6 +847,10 @@ Lukas Zorich <lukas.zorich@gmail.com>
 Luke Peterson <hazelnusse@gmail.com>
 Luv Agarwal <agarwal.iiit@gmail.com>
 Maciej Baranski <getrox.sc@gmail.com>
+Maciej Skorski <maciej.skorski@gmail.com> Maciej Skorski <31315784+maciejskorski@users.noreply.github.com>
+Maciej Skorski <maciej.skorski@gmail.com> Maciej Skorski <maciej.skorski@gmail.com>
+Maciej Skorski <maciej.skorski@gmail.com> Maiej Skórski <maciej.skorski@gmail.com>
+Maciej Skorski <maciej.skorski@gmail.com> maciejskorski <maciej.skorski@gmail.com>
```

Comment:
If you set your name as intended in git locally and then squash the commits then you won'y need all of these entries.

It isn't necessary until right before merge but the commits will need to be squashed because we don't need 40 commits for this change.

Reply:
You mean that the main Sympy repo will not need it, once you merge via a squashing commit?
My fork still needs the mapping due to the commit history I think.
In fact, you have repeated entries in the main branch already, so I understand why you point this out.

Reply:
The SymPy repo does not merge with squashing commits. See gh-18548

You will need to manually squash these commits yourself. Once that is done only the name/email used for the squashed commit needs to be in .mailmap.